src/pybind/mgr/dashboard/frontend/protractor.conf.js

```diff
@@ -53,6 +53,7 @@ const config = {
 
 config.onPrepare = async () => {
   await browser.manage().timeouts().implicitlyWait(config.implicitWaitTimeout);
+  await browser.waitForAngularEnabled(false);
```

Comment:
Is that a mistake?

Reply:
Unfortunately no.
This line was on the logs test file and was being loaded globally when we run the tests, I just moved it to a better location.
If I remove it, all tests will fail. I think its a problem in the login process.
I'm thinking of trying to fix it in a later PR.

Reply:
Okay, so whenever `Logs` has been tested, after that test, the synchronization/waiting might have been disabled. But before `Logs` were tested, all suites had `waitForAngularEnabled(true)`. But by putting the configuration in `protractor.conf.js`, they'll always be `false` for all tests, which could make those tests more flaky which had it set to `true` before.

Reply:
since it was outside of the test class, I think it was being loaded for all the tests, not only the ones after logs.
At least I was not able to run any test suite, besides the logs one.